server/datastore/mysql/hosts.go

```diff
@@ -1631,7 +1640,10 @@ func matchHostDuringEnrollment(ctx context.Context, q sqlx.QueryerContext, isMDM
 	//	args = append(args, uuid)
 	// }
```

Comment:
@mna Do we need this commented here? AFAICS the below query that uses serial number instead of UUID is working correctly for MDM (all macOS devices have and report unique UUIDs and serial numbers)

Reply:
Yeah I think it can be safely removed, doesn't seem like we'll need this anytime soon (it's been there for quite a while already).